src/functions/operatorname.js

```diff
@@ -30,37 +30,44 @@ defineFunction({
             let mode = "";
 
             // Consolidate Greek letter function names into symbol characters.
-            const temp = html.buildExpression(group.value.value, options, true);
+            const temp = html.buildExpression(
+                group.value.value, options.withFontFamily("mathrm"), true);
 
             // All we want from temp are the letters. With them, we'll
             // create a text operator similar to \tan or \cos.
             for (let i = 0; i < temp.length; i++) {
```

Comment:
(optional) You could change this to `for (const child of temp) {` if we have a preference for that. It makes it more up front that the index is not used anywhere else.